backend/persistent/PlotlyAPI.js

```diff
@@ -26,15 +26,34 @@ export function PlotlyAPIRequest(relativeUrl, {body, username, apiKey, accessTok
     });
 }
 
-export function newDatacache(payloadJSON, type) {
+export function newDatacache(payloadJSON, type, requestor) {
     const form = new FormData();
     form.append('type', type);
     form.append('payload', payloadJSON);
     const body = form;
 
-    return fetch('https://plot.ly/datacache', {
+    const users = getSetting('USERS');
+    const user = users.find(
+        u => u.username === requestor
+    );
+    const apiKey = user.apiKey;
+    const accessToken = user.accessToken;
+
+    let authorization = '';
```

Comment:
I'm OK with this for now, but please log an issue re: 🐫ing it (it copies the code in `PlotlyAPIRequest` pretty closely).